sass/_theme_layout.sass

```diff
@@ -609,4 +609,6 @@ h4
 p
   font-size: 1.6rem
   line-height: 2.4rem
-  margin-bottom: 2.4rem
\ No newline at end of file
+  margin-bottom: 2.4rem
+p, li, h1, h2, h3, h4, h5, h6
+  max-width: 40em
```

Comment:
In teoria, le nuove LG di design utilizzeranno un font-size di base di 18px anziché 16px su schermi grandi, per cui questa cosa potrebbe essere invalidata. Per ora va bene così, però andrò a creare una issue a riguardo.

Reply:
io sarei per mettere 18px anche sul tema (cmq 40em va bene a ogni size ;)

Reply:
Sulla lunghezza complessiva delle righe c'è anche questa discussione: https://github.com/italia/designers.italia.it/issues/80